modules/install/install_rke2_master_eip.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
dont create a new file for this, use the existing ones
refactor those to accept your needs parametrized

Reply:
thanks done!